Assets/Scripts/light_variable_intensity.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class light_variable_intensity : MonoBehaviour
```

Comment:
Another war crime!
Types like struct, class, enum should be in Pascal case `LightVariableIntensity `